pkg/operator/configobservation/serviceca/observeserviceca.go

```diff
@@ -0,0 +1,62 @@
+package serviceca
+
+import (
+	"k8s.io/apimachinery/pkg/api/equality"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/configobservation"
+	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/operatorclient"
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+)
+
+const (
+	serviceCAConfigMapName = "service-ca"
+	serviceCABundleKey     = "cabundle.crt"
```

Comment:
OAuth server and SA stuff uses `ca.crt`.  I also see lots of `ca-bundle.crt`.  We should probably pick one of those?

Reply:
> OAuth server and SA stuff uses `ca.crt`. I also see lots of `ca-bundle.crt`. We should probably pick one of those?

ca-bundle.crt

Reply:
https://github.com/openshift/service-ca-operator/pull/27